core/src/main/java/tc/oc/pgm/community/command/ModerationCommand.java

```diff
@@ -809,7 +810,25 @@ private void sendWarning(MatchPlayer target, String reason) {
         TextComponent.builder().append(WARN_SYMBOL).append(titleWord).append(WARN_SYMBOL).build();
     Component subtitle = formatPunishmentReason(reason).color(TextColor.GOLD);
 
-    target.showTitle(title, subtitle, 5, 200, 10);
+    // Legacy support - Displays a chat message instead of title
+    if (ViaUtils.getProtocolVersion(target.getBukkit()) <= ViaUtils.VERSION_1_7) {
```

Comment:
We are using this `if` statement a lot. Do you mind a quick refactor? Add a `MatchPlayer#isLegacy` method.

Reply:
Sure thing 👍